src/main/webapp/events.html

```diff
@@ -27,9 +27,18 @@
 				<a id="login-url">Login</a>
 			</div>
 		</div>
-        <div id="my-events-and-add-buttons"></div>
+    <div id="my-events-and-add-buttons"></div>
 		<div id="events-search-area"></div>
 		<div id="edit-form-area" class="hide-edit-modal"></div>
+		<div id="footer">
```

Comment:
Can you build this in JS that gets constructed when the DOM is available? Otherwise if you change the footer you'll have to change it everywhere.